modules/wpa_screenshot_capture/src/Plugin/CompareResponse/FileSizeScreenshotVarianceCompareResponse.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\wpa_screenshot_capture\Plugin\CompareResponse;
+
+/**
+ * The response that indicates the variance threshold for a screenshot response.
+ */
+class FileSizeScreenshotVarianceCompareResponse extends ScreenshotVarianceCompareResponse {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getId() {
+    return 'wpa_file_size_screenshot_variance_compare_response';
+  }
+
+}
```

Comment:
This appears to be pointless right now, but it will make more sense once the imagemagick one gets fleshed out as we're gonna have to figure out at that point how to handle competing buttons and it's probably gonna have to happen in the `renderPreview()` method.